superset/templates/superset/dashboard.html

```diff
@@ -22,4 +22,5 @@
   <div id="grid-container" class="slice-grid gridster"></div>
 
 </div>
+<input type="hidden" name="csrf_token" id="csrf_token" value="{{ csrf_token() }}">
```

Comment:
i don't think we need to add this here since this template extends from basic.html which includes this input. https://github.com/airbnb/superset/pull/2552/files#diff-9d6afb3a3cc20a67b44bdfeebe1f90bfR1